### PR TITLE
Dodanie Layout ( Ramy ) strony

### PR DESCRIPTION
**Pull request by tomaszprc**

### CODE REVIEW

**Comment by tomaszprc** [comment]
Jedynie taki mam błąd, mam z tym coś zrobić?

image

File: src/App.tsx [hunk 1]
```diff
@@ -11,21 +11,12 @@ const App: FC<Props> = () => {
     return (
         <>
             <GlobalStyle />
-            <ContentWrapper>
+            <Container>
```
Comment (Andrzej-Marek) [inline src/App.tsx:14]: React always have to have only one child. So wrap all inside Container in empty <></>

File: src/components/Container/Container.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import React, { FC } from "react";
+import { media } from "@/shared/utils";
+import styled from "styled-components";
+
+interface OwnProps {
+    children: JSX.Element
```
Comment (Andrzej-Marek) [inline src/components/Container/Container.tsx:6]: Don't pass children prop here, it's already in FC ( react ) type

File: src/components/Container/Container.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+const ContainerWrapper = styled.div`
+    max-width: 100%;
+    margin-right: auto;
+    margin-left: auto;
+    padding: 0 20px;
+
+    ${media.tablet`
+         max-width: 750px;
```
Comment (Andrzej-Marek) [inline src/components/Container/Container.tsx:27]: To increase UX just use padding: 10px 0 ( and more on bigger screens ) and set the width to 95vw ( and small or bigger screen ). Test how to resize flow works

Comment (tomaszprc) [reply src/components/Container/Container.tsx:27]: Ok, I removed max-width from css and I created this one
image

I set smaller width on bigger screens. Is it correct?

Comment (tomaszprc) [reply src/components/Container/Container.tsx]: To quote your message : " padding: 10px 0 " , it is style for padding-y (top,bottom) I think you thought padding-x (left and right). I will set the padding-y later in others components

File: src/App.tsx [hunk 4]
```diff
@@ -1,31 +1,29 @@
...
-        <>
+        <MainWrapper>
             <GlobalStyle />
-            <ContentWrapper>
-                <p>Welcome to my starter!</p>
-                <PrimaryButton text="Hello world" width="150px" />
-            </ContentWrapper>
-        </>
+            <Container>
+                <>
```
Comment (Andrzej-Marek) [inline src/App.tsx:16]: Why you use <></> if you have only one child?

File: src/App.tsx [hunk 5]
```diff
@@ -1,31 +1,29 @@
...
 
-const ContentWrapper = styled.div`
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    flex-direction: column;
-
+const MainWrapper = styled.div`
     height: 100vh;
+    background: #E5E5E5;
```
Comment (Andrzej-Marek) [inline src/App.tsx:26]: use colors from theme ( check primary button example )

File: src/components/Container/Container.tsx [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+    margin-right: auto;
+    margin-left: auto;
+    padding: 0 20px;
+
+    ${media.laptop`
+        width: 85vw;
+    `}
+
+    ${media.laptopL`
+        width: 75vw;
```
Comment (Andrzej-Marek) [inline src/components/Container/Container.tsx:29]: You don't have max width set.  set max width to 1200 px

File: src/components/index.ts [hunk 7]
```diff
@@ -1,3 +1,7 @@
 // Buttons
 
 export { default as PrimaryButton } from "./Buttons/PrimaryButton/PrimaryButton";
+
+//Container
```
Comment (Andrzej-Marek) [inline src/components/index.ts:5]: Add comments when more than one it's importing from the folder. Like you have buttons folder and them more folders. If it is only one folder, then you don't have to type a comment

**Review by Andrzej-Marek — CHANGES_REQUESTED** [review]

**Review by Andrzej-Marek — APPROVED** [review]

**Pull request merged**